core/controllers/suggestion.py

```diff
@@ -139,6 +161,68 @@ def put(self, target_id, suggestion_id):
         self.render_json(self.values)
 
 
+class ReviewableSuggestionsHandler(base.BaseHandler):
+    """Provides all suggestion which can be reviewed by the user for a given
+    suggestion type.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.can_access_admin_page
+    def get(self, target_type, suggestion_type):
+        """Handles GET requests."""
+        try:
+            _check_valid_suggestion_type_and_target_type(
+                target_type, suggestion_type)
+
+            suggestions = suggestion_services.get_reviewable_suggestions(
+                self.user_id, suggestion_type)
+
+            if target_type == suggestion_models.TARGET_TYPE_EXPLORATION:
+                target_id_to_details = (
+                    opportunity_services.get_opportunities_by_ids(
+                        [s.target_id for s in suggestions]))
+                self.render_json({
+                    'suggestions': [s.to_dict() for s in suggestions],
```

Comment:
Is there any reason to have two keys here, rather than combining into one? (I'm assuming the list of suggestions has the same number of elements as keys in the target_ids_to_details dict, but please correct me if I'm wrong about that.)

Reply:
Nope, there can be multiple suggestions for one exploration, so they can use a single element of target_ids_to_details to represent themselves.